scripts/migrate.py

```diff
@@ -107,7 +115,26 @@ def main():
 
     # Capture all logs for all migrations
     final_logs = {}
-    final_logs['users'] = import_users(old_db, new_db)
+    steps = [s.strip().lower() for s in (args.steps or '').split(',') if s.strip()]
+
+    try:
```

Comment:
Do not use huge `try` blocks
Use independent try block if required.
Do not use them in this case.
If a steps fails we domn't want to continue. We want the error to be raised